torch/_inductor/codegen/wrapper.py

```diff
@@ -1207,7 +1195,7 @@ def write_wrapper_decl(self):
                     c10::optional<c10::string_view> optional_string;
                     c10::optional<at::Layout> optional_layout;
                     c10::optional<at::Tensor> optional_tensor;
-                    torch::List<c10::optional<at::Scalar>> optional_list;
+                    c10::List<c10::optional<at::Scalar>> optional_list;
```

Comment:
dumb question - why are we switching to ```c10::List```?

Reply:
It was coming my other include file change. Just for uniformity.